customization.cfg

```diff
@@ -84,6 +84,12 @@ _cpusched=""
 # For advanced users.
 _compiler=""
 
+# Clang LTO mode - options are "full" or "thin".
```

Comment:
We should also explicitly state that it's only available when choosing `llvm` as a compiler. Some of us don't know that `llvm` or `clang` in our case it's the same thing.